packages/itwinui-css/backstop/tests/assets/demo.css

```diff
@@ -61,12 +61,12 @@ body {
 
```

Comment:
Is it okay to leave `margin: 11px 0` for `hr`

Reply:
Updated to use 12px here and in a couple other places that were missed.